fedocal/templates/default/view_meeting.html

```diff
@@ -54,10 +54,16 @@ <h2 class="orange">{{ _('Meeting "%(name)s"', name=meeting.meeting_name) }}</h2>
     {% endmacro %}
     <p>{{ _('Meeting organized by %(managers)s', managers=get_managers(meeting.meeting_manager)) }}</p>
 
-    <a href="{{ url_for('ical_calendar_meeting', meeting_id=meeting.meeting_id) }}"
+    <a id="ical-meeting-export" href="{{ url_for('ical_calendar_meeting', meeting_id=meeting.meeting_id) }}"
        title="{{ _('Add this meeting to your agenda') }}">
        {{ _('iCal export') }}
     </a>
+    <span>
+        <label for="ical-meeting-export-reminder-toggle">
+            <input id="ical-meeting-export-reminder-toggle" type="checkbox">
+            <em>{{ _('with reminder') }}</em>
```

Comment:
What will we have in the dropdown? Reminder before 1 day, 1 hour and 5 minutes? I think it will just complicate things in terms of UI. We can ask mizmo to help out with this decision :)